server/controller/feedsController.js

```diff
@@ -5,7 +5,10 @@ class FeedsController {
   static async getAllFeeds(req, res) {
     try {
       const userId = req.user.userId;
-      const feeds = await pool.query(queries.getAllFeeds, [userId]);
+      const { offset } = req.query;
```

Comment:
do both on the same line
`const {offset, limit} = req.query`

Reply:
Ok sir.I have made this change